CMakeLists.txt

```diff
@@ -1417,6 +1417,14 @@ if(CONFIG_BUILD_OUTPUT_ADJUST_LMA)
     )
 endif()
 
+list(APPEND remove_sections ".comment" "COMMON")
```

Comment:
please use a `set(remove_sections ...)` for first occurrence unless there is a desire for allowing externally pre-defined sections (in which case such feature should be documented and reasoned).

Using a `list(APPEND ...)` bears the risk of unexpectedly picking up already defined var, and if that happens it can be hard to debug.